frontend/src/components/common/TimezoneSelect/TimezoneSelect.tsx

```diff
@@ -51,7 +51,16 @@ export default function TimezoneSelect(props: TimezoneSelectorProps) {
       includeInputInList
       openOnFocus
       renderInput={params => (
-        <TextField {...params} helperText={t('Timezone')} size="small" variant="outlined" />
+        <TextField
+          {...params}
+          helperText={t('Timezone')}
+          size="small"
+          variant="outlined"
+          inputProps={{
+            ...params.inputProps,
+            'aria-label': t('Timezone'),
```

Comment:
Same comment as to why this is needed. Can you double check if MUI is not setting this and we're just missing some param?